libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -540,16 +561,16 @@ interface BlockingBreezServices {
    Payment send_spontaneous_payment(string node_id, u64 amount_sats);
 
    [Throws=SdkError]
-   ReceivePaymentResponse receive_payment(ReceivePaymentRequest req_data);
+   ReceivePaymentResponse receive_payment(ReceivePaymentRequest request);
 
    [Throws=SdkError]
-   LnUrlPayResult pay_lnurl(LnUrlPayRequestData req_data, u64 amount_sats, string? comment);
+   LnUrlPayResult pay_lnurl(u64 amount_sats, string? comment, LnUrlPayRequestData request_data);
```

Comment:
NIT: IMO the reordering of LNURL args wasn't needed. They were ordered by importance:
- `req_data`: the data returned by `parse` in the first step of LNURL
- `amount`
- `comment` which was optional

Reply:
I see, it wasn't clear from the SDK implementation. 
I will revert these changes.

Reply:
Done e48902088783449e148bba6c6b89528c0fa21451